submissions/description/server_security_misconfiguration/directory_listing_enabled/non_sensitive_data_exposure/guidance.md

```diff
@@ -0,0 +1,7 @@
+# Guidance
+
+Provide a step-by-step walkthrough with a screenshot on how you exploited the vulnerability. This will speed triage time and result in faster rewards.
```

Comment:
Should this generic guidance be included in this template? The others appear to be more specific.

Reply:
As discussed, I'll leave it for now and note it down for changes in the future as there is no guidance doc currently up the tree @vortexau